### PR TITLE
docs(color): Add new color palette, link to old palette

### PR DESCRIPTION
**Pull request by glynnis**
https://jira.rax.io/browse/FRMW-259

New color palette, ready for review!

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### CODE REVIEW

**Comment by glynnis** [comment]
screen shot 2015-10-22 at 9 02 31 am
screen shot 2015-10-22 at 9 02 48 am
screen shot 2015-10-22 at 9 02 41 am

**Comment by CITguy** [comment]
So are we no longer displaying the color hex values?

**Comment by CITguy** [comment]
All of the palettes use the "500" value as the background for their title. Should the gray scale do the same?

**Comment by glynnis** [comment]
@CITguy Consulted with @ElementE, and he sees the gray scale as its own special snowflake, so calling out a primary there isn't necessary.

Hex values aren't being displayed since variables are provided via the Framework, and we want to encourage projects to use variables rather than hard-coding colors at the project level (so there's some degree of future-proofing).

**Comment by glynnis** [comment]
Thinking more about it, and wondering if designers will still need hex values to be present :confused: 

Maybe we show both? I just want to be sure it's clear to devs that the variables are the only thing they should use in LESS, but I suppose we could catch that kind of thing with a linter?

**Comment by CITguy** [comment]
There might be a possibility to check that colors are used only for variables. `lesshint` doesn't have support for that yet, though.

File: src/rxApp/rxApp.less [hunk 1]
```diff
@@ -481,7 +481,7 @@
 
         & > .rx-notifications {
             position:fixed;
-            z-index: 5;
+            z-index: 20;  // accommodate rxSelect .fake-select class
```
Comment (CITguy) [inline src/rxApp/rxApp.less]: This change was made in another PR. You might want to rebase.

**Comment by glynnis** [comment]
Updated this branch and removed the old screen shots. Not including "before" shots since a lot of this is new content.

Added hex codes, content, and color usage guide.

screen shot 2015-10-27 at 3 31 47 pm
screen shot 2015-10-27 at 3 31 53 pm
screen shot 2015-10-27 at 3 32 19 pm

File: demo/assets/demo.less [hunk 2]
```diff
@@ -133,6 +133,10 @@ table.component-attributes {
     }
 }
 
+.page-content {
```
Comment (glynnis) [inline demo/assets/demo.less]: Added this in because of some issues that came in from `master` with titles and floats.

File: demo/styles/buttons.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<rx-page title="'Buttons &amp; Links'" class="stylguide">
+<rx-page title="'Buttons &amp; Links'" class="styleguide">
```
Comment (glynnis) [inline demo/styles/buttons.html]: Unrelated change -- found a typo. Sneaking it in with this PR.

File: demo/styles/color-1-0.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
+<rx-page title="'Color'" class="styleguide">
```
Comment (glynnis) [inline demo/styles/color-1-0.html]: Should I change this title to something like "1.0 Color Palette (Deprecated)"?

**Comment by glynnis** [comment]
Removing "DO NOT MERGE" as this is ready for review.

**Comment by glynnis** [comment]
Screenshots updated to reflect that I merged in @CITguy's branch.

**Comment by CITguy** [comment]
@glynnis please rebase

File: demo/styles/color.html [hunk 5]
```diff
@@ -1,101 +1,322 @@
...
   <p>
-    Color is an essential part of communicating with your Encore users.
-    Certain colors in Encore are reserved for particular actions of events.
+    If you use colors from this palette in your project,
+    <strong>only use the variable names provided, NOT hex codes</strong>.
+    The LESS variables for colors ship by default with the Framework and are
+    available at the project level. Using them ensures consistency across Encore
+    as colors shift or change over time, and will save you from having to revise
+    hard-coded hex codes in your code the future. Hex codes are provided here
+    for reference only.
```
Comment (CITguy) [inline demo/styles/color.html]: Missing closing `</p>`

File: src/rxApp/common.less [hunk 6]
```diff
@@ -2,6 +2,7 @@
  * Common
  */
 @import url("https://fonts.googleapis.com/css?family=Roboto:400,100,100italic,300,300italic,400italic,700,700italic");
+@import url("https://fonts.googleapis.com/css?family=Roboto+Mono:400,100,100italic,300,300italic,400italic,700,700italic");
```
Comment (CITguy) [inline src/rxApp/common.less]: Yay! "Roboto Mono"

File: demo/styles/color-1-0.html (hunk 4 above)
Comment (ssalinas24) [inline demo/styles/color-1-0.html]: @glynnis I would agree with that.

File: demo/styles/color.html [hunk 7]
```diff
@@ -1,101 +1,322 @@
 <rx-page title="'Color'" class="styleguide">
   <p>
-    Color is an essential part of communicating with your Encore users.
-    Certain colors in Encore are reserved for particular actions of events.
+    If you use colors from this palette in your project,
+    <strong>only use the variable names provided, NOT hex codes</strong>.
+    The LESS variables for colors ship by default with the Framework and are
+    available at the project level. Using them ensures consistency across Encore
+    as colors shift or change over time, and will save you from having to revise
```
Comment (ssalinas24) [inline demo/styles/color.html]: @glynnis "revise hard-coded hex codes in your code the future."
could probably change to:
"revise hard-coded hex codes in the future."

File: demo/styles/color.html [hunk 8]
```diff
@@ -1,101 +1,322 @@
...
         <ul class="list">
+          <li>Used for links and icons that deal explicitly with create and add actions.</li>
           <li>
-            Used for links and icons that deal explicitly with create and add
-            actions.
-          </li>
-          <li>
-            Should not be used to communicated "active" or "success"
-            (since those are states, not actions a user can take)
+            Should not be used to communicated "active" or "success" (since those are
```
Comment (ssalinas24) [inline demo/styles/color.html]: @glynnis "communicated" should be "communicate".

File: demo/styles/color.html [hunk 9]
```diff
@@ -1,101 +1,322 @@
...
+            states, not actions a user can take
```
Comment (ssalinas24) [inline demo/styles/color.html]: @glynnis Also need a ")" at the end of the line.

File: demo/styles/color.html [hunk 10]
```diff
@@ -1,101 +1,322 @@
...
+      <div class="color shade-300 dark-text">
+        <div>Gray - Disabled<br />300</div>
+        <div>@gray-300 <small>#E0E0E0</small></div>
       </div>
-      <div class="swatch-body">
-        Used to communicated a button, link, or element is disabled.
+      <div class="detail">
+        <ul class="list">
+          <li>
+            Used to communicated a button, link, or element is disabled.
```
Comment (ssalinas24) [inline demo/styles/color.html]: @glynnis "communicated" should be "communicate" here also.

File: demo/styles/color-1-0.html [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+    <div class="swatch">
+      <div class="swatch-heading swatch-green">#24b56a - @green</div>
+      <div class="swatch-body">
+        <ul class="list">
+          <li>
+            Used for links and icons that deal explicitly with create and add
+            actions.
+          </li>
+          <li>
+            Should not be used to communicated "active" or "success"
```
Comment (ssalinas24) [inline demo/styles/color-1-0.html]: @glynnis "communicated" should be "communicate"

File: demo/styles/color-1-0.html [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+        associated with success, failure, warning, or error.
+      </div>
+    </div>
+
+    <div class="swatch">
+      <div class="swatch-heading swatch-disabled-gray">
+        #dededd - @disabled-gray
+      </div>
+      <div class="swatch-body">
+        Used to communicated a button, link, or element is disabled.
```
Comment (ssalinas24) [inline demo/styles/color-1-0.html]: "communicated" should be "communicate that" here as well.

**Comment by glynnis** [comment]
Responded to feedback and rebased/squashed.

**Comment by glynnis** [comment]
More :eyes: :eyes: :eyes:  plz

**Comment by CITguy** [comment]
LGTM

**Comment by ssalinas24** [comment]
LGTM

**Comment by bry4n** [comment]
Woot Color 2.0!

LGTM
